config-generators/cosmosdb_nosql-commands.txt

```diff
@@ -12,4 +12,6 @@ add Earth --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.earth" -
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:create" --fields.include "id" --fields.exclude "name"
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:read" --fields.include "id,type" --fields.exclude "name"
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:update" --fields.exclude "*"
-update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "authenticated:create,read,update,delete"
\ No newline at end of file
+update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "authenticated:create,read,update,delete"
+add Sun --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.sun" --permissions "anonymous:create,update,delete" --graphql true
+update Sun --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:read" --fields.exclude "*"
```

Comment:
here, the command is used to add `fields.exclude`, but in the config file it's showing under `fields.include`

Reply:
Yes, this caused the integration tests to fail as well. I've fixed this, thank you for catching it!